server/app/onboarding.go

```diff
@@ -85,6 +85,10 @@ func (a *App) createWelcomeBoard(userID, workspaceID string) (string, error) {
 
 	blocks = model.GenerateBlockIDs(blocks, a.logger)
 
+	if updateFileIDsErr := a.CopyCardFiles(onboardingBoardID, workspaceID, blocks); updateFileIDsErr != nil {
```

Comment:
There will likely be a linter error here... error vars should start with "err".

Reply:
The linter didn't fail but I've renamed it anyways.

Reply:
Thanks.  We have not enabled the linter for CI yet.